types/ApiResponses.ts

```diff
@@ -123,6 +129,10 @@ export interface CreateEventResponse extends ApiResponse {
   event: PublicEvent;
 }
 
+export interface AddEventFeedbackResponse extends ApiResponse {
```

Comment:
Nit: be consistent about language. There's "submit feedback" "provide feedback" "add feedback" so far.